docs/python/python-quick-start.md

```diff
@@ -0,0 +1,112 @@
+---
+Order: 1
+Area: python
+TOCTitle: Quick Start
+ContentId: c7134463-4fdd-4674-8685-77c94472902c
+PageTitle: Quick Start Guide for Python in VS Code
+DateApproved: 1/17/2024
+MetaDescription: A quick start guide to get you up and coding with the Python extension in Visual Studio Code.
+MetaSocialImage: images/tutorial/social.png
+---
+
+# Quick Start Guide for Python in VS Code
+
+The Python extension makes Visual Studio Code an excellent Python editor, works on any operating system, and is usable with a variety of Python interpreters.
+
+Get started by installing:
+-	[VS Code](https://code.visualstudio.com/)
+-	[A Python Interpreter](/docs/python/python-tutorial.md#_install-a-python-interpreter) (any [actively supported Python version](https://devguide.python.org/#status-of-python-branches))
+-	[Python extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python) from the VS Code Marketplace
```

Comment:
Should this be spelled out as well? 
```suggestion
-	[Python extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python) from the Visual Studio Code Marketplace
```

Reply:
Not really. We spell it out only on the first occurrence on a page.